webapp/src/ts/modules/contacts/contacts.component.ts

```diff
@@ -356,9 +356,10 @@ export class ContactsComponent implements OnInit, AfterViewInit, OnDestroy {
 
     let docIds;
     if (options.withIds) {
-      docIds = this.contactsList.map((item) => {
-        return item._id;
-      });
+      docIds = this.contactsList.map((item) => item._id);
+      if (this.usersHomePlace && !docIds.includes(this.usersHomePlace._id)) {
+        docIds.push(this.usersHomePlace._id);
+      }
     }
```

Comment:
This function is too long and getting complex, I'm wondering why sonar didn't warn

Alternatively, this can be extracted to a function that returns docIds.